code/development-env/config/local/catalogue_config.json

```diff
@@ -0,0 +1,3 @@
+{
+  "available": true
```

Comment:
* Is this a config map, i.e. will there be a corresponding PR in datalab-k8s-manifests to inject the config map JSON in the right place?
* Given that there's only one parameter, would it have been better to do this via environment variable, e.g. https://github.com/NERC-CEH/datalab-k8s-manifests/blob/master/templates/datalab/datalab-api-deployment.template.yml#L52?  This would have the added benefit of then being accessible synchronously rather than asynchronously, e.g. https://github.com/NERC-CEH/datalab/blob/master/code/workspaces/client-api/src/auth/authServiceClient.js#L36

Reply:
* You are correct, there should be a corresponding PR in the manifests repo that I will add shortly!
* In a separate discussion we covered that:
  * It would have been best to do this as a simple environment variable; I aimed for a solution that should be robust to future complexity with the catalogue configuration but this was premature. 
  * As this work has already been done and it should be robust to future complexity in configuration, we should leave this as is rather than changing to use an environment variable.